### ISSUE TITLE
Anyone have TT working or what am I doing wrong.

### ISSUE THREAD

**Issue by MervHarvey** [jbrunken/ActiveForumsTapatalk#4]
Okay this is a two edge sword. 1) I'm learning my way around here, not sure if this is the correct place to be posting.

2) I had a go at running TT some time ago. All that was returned was code. I was using a WP 8 and also one of my moderators results were the same with an android phone.

Now with this current release we having another go. This time we see on the Home page six tiles (well I do on WP 8) Subscribed Forums, Subscribed Topics, Participated, Unread, Timeline and Forums. The only tile that displays anything is Forums, on tap (Forum Groups) on tap Group (Subforums) on tap Subforum (discussions) but there are no topics or posts.

There is a create new topic button (+) that works and I can create a new topic. New topic is written to the forum but is not available on TT.

What's wrong? As far as I can tell I have all settings set.
##

**Comment by frozendnn** [comment]
I am having some issues with TT as well. I see that Jason's website is working fine so I assume something is wrong with our settings. 

Dont know how to fix it though. 

Sent from Yahoo Mail on Android

**Comment by jbrunken** [comment]
@MervHarvey You are posting in the correct spot.  Are you using the Official Tapatalk client, or the 3rd party one?  If you're using the 3rd party one, I'd switch to the official client and see if that helps.   Tapatalk uses the XMLRPC protocol for its communication and it's very finicky.  If things aren't just right on both ends, it tends to have issues.  I know that the current release works well with iOS and Android, but I don't have any access to a Windows Phone, so I don't have any way to test with that.  

@frozendnn Have you installed the latest version of the plugin that I posted this week?

**Comment by frozendnn** [comment]
Yes, I have. But for some reason talktalk says could not contact with server. Through tapatalk account, it says code 200 error. 

Could be permission issues? Also tapatalk does not create an tables in DB?

Sent from Yahoo Mail on Android

**Comment by MervHarvey** [comment]
I can only assume that I'm using the Official Tapatalk client. It is the only one listed in the windows store. Ver 1.0.5.0. Checking the app on my phone. About: name - Tapatalk.WP8; Version 1.0.5.0; Support - support.tapatalk.com

**Comment by MervHarvey** [comment]
On tapatalk.com (Basic Tab) there is Tapatalk API Key. I don't see anywhere in the AF TT settings where this key is required.

TT says that it is not required in some instances. YES/NO?

**Comment by jbrunken** [comment]
@MervHarvey You don't need an API key for our plugin.   I had a friend with a Windows phone download the app and give it a quick try on my site.  We didn't test it in great details, but on the surface it seemed to work fine.

How do you have your permissions setup on your forum.   I'm wondering if this might be an issue between the "read" and "view" permissions.  If you don't have those setup correctly, it may not let you view the posts.  This could explain it.   I'll have to look into it tonight and tell you how I have mine setup.

@frozendnn I would suggest going through the Tapatalk installation steps one by one and making sure you have everything setup properly.    Look at every setting carefully to make sure that you don't have a typo somewhere.

**Comment by frozendnn** [comment]
I want to do that but since the DNN upgrade I cannot access module settings. JavaScript errors. Nothing is easy for me. 

Uncaught TypeError: Object [object Object] has no method 'sortable' dnn.dragdrop.js?cdv=404:191
Uncaught TypeError: Object [object Object] has no method 'dialog' dnn.jquery.js?cdv=404:90

Any idea?

**Comment by jbrunken** [comment]
Try clearing your browser cache and then restarting it.  You may have old .js files cached.  If that doesn't work, I'm not sure.   You'll have to ask the dnn team.

**Comment by MervHarvey** [comment]
JB Could be I have most forums Public. View, Read Create and Reply,  All Users, Administrator, Registered Users and Forum Moderators with unregistered users moderated.

Maybe this too. Have a Main Menu item Forums that redirects to the General Forums page.

**Comment by MervHarvey** [comment]
Made All Users View only and turned off Moderation. Made no diff. As a side note. What is the difference between View and Read?

**Comment by jbrunken** [comment]
If you have both view and read on, that should be enough.  I'm not sure what to tell you at this point.  If you want to give me the name of your forum, I can test it myself later using iOS & Android clients.

**Comment by MervHarvey** [comment]
@ JB If you search for RV Supertramp you will find it.

**Comment by jbrunken** [comment]
It's not working in iOS either so there is definitely something going on with your setup.   BTW, the AF tapatalk plugin doesn't currently support anonymous posting.

Oh, and the difference between "View" and "Read" is this:  "View" gives users the ability to see topics in the topic list.  "Read" gives users the ability to actually open the topic and read it.

**Comment by MervHarvey** [comment]
Thanks. I'll talk to the guys over at Tapatalk. When I originally tested I set up the account at TT with a test domain. At that time I believe it could not be edited, now it can and I have done that with TT pointing to the production site.

Could be something there. Will see if they can help. View/Read got it.

**Comment by MervHarvey** [comment]
This is what they said.

"The plugin of your forum was developed by third part developer. Please report any issue to them or where you download the module. "

Yipee.

**Comment by MervHarvey** [comment]
Maybe this is the cause. I have a Forums page which is a main menu item. It redirects to a child page General Forums.  Might try putting a copy on the Forum page.

**Comment by frozendnn** [comment]
12/15/2013 11:43:25 PMHTTP Error Code 404 Page Not Found  website  TabId PortalAlias website.com OriginalUrl /aft628/mobiquo.ashx Url http:// 

TabId:
PortalAlias:website.com
OriginalUrl:/aft628/mobiquo.ashx
Url:http://website.com/aft628/mobiquo.ashx
UserAgent:Mozilla/5.0 Firefox/3.5.6 3.0.0.beta
HostAddress:50.17.254.140
HostName:50.17.254.140
Server Name: ds00796

---

Don't know what's happening but I am getting the above error. Does it matter that my website urls are now extension less (DNN Adavanced URLs)?

**Comment by MervHarvey** [comment]
@jbrunken  I had a bit of a shot at the guy over at TT for his response. He came back with this.

"I registered a test account on the forum and confirmed topic can not be listed and here is the plugin error message I got:

Could not find stored procedure 'activeforumstapatalk_Forum_Permissions'.

Please report the issue to plugin developer. "

**Comment by jbrunken** [comment]
@frozendnn It's possible that the new URL management stuff in DNN 7.1 may be interfering with the Tapatalk handler.  I still haven't had a chance to upgrade any of my systems to 7.1 so I'm not able to test this yet.

@MervHarvey You use an object prefix on your database don't you?   If so, the stored procedure name should include the prefix (which it apparently doesn't).  This would definitely stop it from working.

**Comment by jbrunken** [comment]
@MervHarvey I think I know what I did wrong.  I'll post a new build as soon as I get a chance.  Unfortunately, it probably won't be until tomorrow night.

**Comment by MervHarvey** [comment]
@jbrunken I had a look and all the activeforumstapatalk_...... Procedures include the prefix.

**Comment by jbrunken** [comment]
@MervHarvey still working on the fix.  I think I've about got it done.  Just trying to get it tested.

@frozendnn I confirmed that DNN 7.1 is blocking the tapatalk handler.  I haven't figured out how to fix this in the module yet, but there is a temporary work around.   If you're not using Evoq (DNN Pro), download this free module to manage your url replacement settings: https://dnnurlmanagement.codeplex.com

Follow the instructions and add and instance of the module to an admin only page.  Load the page and on the "Regular Expressions" tab you will see a entry for "Do Not Redirect:"

Append the following text to the very end of the value that is already in the box:  |mobiquo.ashx

After you've done that, the tapatalk handler should start working again.

**Comment by jbrunken** [comment]
@MervHarvey I just put up a pre-release version that should fix the issue you are having with the plugin.  It's only pre-release because I haven't tested it on more than one system so far, but I did test it on a system using an object qualifier.

**Comment by frozendnn** [comment]
@jbrunken: Thanks! That worked.

**Comment by MervHarvey** [comment]
@jbrunken I posted this earlier on Configuration Questions #3  (Wrong place)

@jbrunken .Thanks JB. Seems to be okay in most areas. Had this error on send Reply. I remember Frozen posting something similar a couple of days ago but can't find it now.

Exception in: domain.com/aft1019/mobiquo.ashx
Data:|?xml version='1.0.?>|methodCall>|methodName>reply_post|/methodName>|params>|param>|value>|string>688|/string>|/value>|/param>|params>|value>|base64>|/base64>|/value>|/param>|param>|value>|base64>U3RvcCBoYW5naW5nLiBUcnkgbm93Lg==|/base64>|/value|/param>|/params>|/methodCall>

....

**Comment by jbrunken** [comment]
@MervHarvey Does that happen every time you post?

**Comment by MervHarvey** [comment]
@jbrunken Yes every time. I think that it maybe Window only, that's what I'm getting from my WP 8. I have a few members testing ATM and one posted a reply from an iPad.

Here's what I have found so far with my WP 8 Nokia 920

1) Everything has to be typed. There is no Talk.
2) Replies throw the error above.
3) Emoticons in posts don't display. HTML does  | img ................. />
4)Viewing a Forum. Forum > Sub forum > Discussions. Returns "No data Found". Unless there is a Pinned and/or Locked Topic.

Also the module says  to use http://domain.com.au when the forum is http://domain.com.au/Forums. Seems to work with either.

**Comment by jbrunken** [comment]
@MervHarvey My hunch is that most of these issues are related to the Windows Phone version of the app.  Item 1 is certainly not something the plugin has anything to do with.   Let me know what you hear from non-WP users.  If they are consistent on the other platforms, I can look into them further.

**Comment by MervHarvey** [comment]
Will do. Thanks for the upgrade anyway. We're getting there.

**Comment by MervHarvey** [comment]
@jbrunken Early reports are that iOS and Android are working. No issues reported.

**Comment by MervHarvey** [comment]
@jbrunken Use reports that links are not hypertext. Member using Android Phone and I found the same on my WP 8.

**Comment by peppertree** [comment]
I can't get it work either. I have added the donotredirect setting. All i can see in the app is a spinning wheel. Are there any logs one can inspect?

Search for dnn-connect in case you want to try it yourself.

**Comment by peppertree** [comment]
bloody me. Did not check the "enable api" checkbox!
